### ISSUE TITLE
Recipes for reproducing "Worst-case forget set in data-wise unlearning"

### ISSUE THREAD

**Issue by puigde** [OPTML-Group/Unlearn-WorstCase#4]
Hi @ljcc0930 @a-F1 :):) 

Not really an issue with the code but an ask. I am trying to reproduce the unlearning evaluations. In appendix B1 you state:
For FT [66], RL [26], EU-k [25], CF-k [25], and SCRUB [39],
the unlearning process takes 10 epochs, during which the optimal learning rate is searched within the range of [10−4, 10−1]. It is not super clear how this should be specified in the main_evalmu.py args since the provided example is for retraining. Also, for the random sets, did you use the same learning rate?

Could you please provide "the recipes" on how to reproduce Table 3 with the current version of the code? A bash or txt file with the launching commands for different methods would be veeery helpful.

I tried with the default rate and scrub but seems that the model is diverging on the unlearning process and does not reproduce.

**Comment by a-F1** [comment]
Thank you for your appreciation and recognition of our work. For selecting the optimal hyperparameters for different unlearning methods, we first perform a grid search on the Random Forget Set and choose the hyperparameters corresponding to the smallest averaging gap (avg.) as the optimal ones. For the same approximate unlearning method on the Worst-Case Forget Set, we use the same values.

We will update the README with commands for different methods as soon as possible. If you have any other questions, please feel free to contact us.

**Comment by a-F1** [comment]
The issue has been closed. If you have any further questions, please feel free to contact us.

**Comment by AKANKSHASINGH233** [comment]
Hi  @ljcc0930 @a-F1 :):)
the command-python main_selmu.py --arch resnet18 --dataset cifar10 --cp_path results/cifar10_resnet/origin/0model_SA_best.pth.tar --select_epochs 2 --num_indexes_to_replace 4500 --unlearn w_retrain --unlearn_steps 2 --theta_lr 1e-3 --save_dir results/cifar10_resnet/data-wise 
followed by 
python main_evalmu.py --arch resnet18 --dataset cifar10 --cp_path results/cifar10_resnet/origin/0model_SA_best.pth.tar --unlearn retrain --num_indexes_to_replace 4500 --unlearn_steps 182 --theta_lr 0.1 --w_path results/cifar10_resnet/data-wise/select_weight.pth.tar --save_dir results/cifar10_resnet/evaluation
yields UA almost same as RA I tried but i am unable to understand why is it reducing?
